spec/integration/crud_spec.rb

```diff
@@ -127,13 +127,13 @@
   describe 'upsert' do
     context 'with default write concern' do
       it 'upserts' do
-        collection.count_documents({}).should == 0
+        collection.count_documents().should == 0
```

Comment:
For all of these instances where you've removed the argument, you can just remove the parentheses entirely (i.e. `collection.count_documents.should == 0`)